artifactory/utils/golang/project/project.go

```diff
@@ -102,10 +118,57 @@ func (project *goProject) PublishPackage(targetRepo, buildName, buildNumber stri
 	if err != nil {
 		return err
 	}
+	// Create the info file if Artifactory version is 6.10.0 and above.
+	version := version.NewVersion(_go.ArtifactoryMinSupportedVersionForInfoFile)
+	artifactoryVersion, err := servicesManager.GetConfig().GetArtDetails().GetVersion()
+	if err != nil {
+		return err
+	}
+	if version.AtLeast(artifactoryVersion) {
```

Comment:
I think that the above if statement means - 
I the min-supported-version is at least the Artifactory version - it seems like it should be the other way around.
It is worth also checking the other info file version comparisons.

Reply:
Per our discussion, I've changed the api.